web/src/settings_notifications.ts

```diff
@@ -77,7 +87,11 @@ function update_desktop_icon_count_display(settings_panel) {
     }
 }
 
-export function set_notification_batching_ui($container, setting_seconds, force_custom) {
+export function set_notification_batching_ui(
+    $container: JQuery,
+    setting_seconds: number,
+    force_custom?: boolean,
```

Comment:
```suggestion
    force_custom = false,
```

to avoid the `=== true` game.